BUILD.md

```diff
@@ -0,0 +1,88 @@
+# Building DLAF
+
+## Build using Spack
+
+Example installation:
+
+`spack install dla-future ^[virtuals=blas,lapack] intel-oneapi-mkl`
```

Comment:
From the formatting I think you want to use a code block (triple backticks) instead of inline code? Same thing for most of the standalone code below.